src/cc1101.cpp

```diff
@@ -443,29 +524,174 @@ void cc1101::CCinit(void) {                                   // initialize CC11
 	cc1101_Deselect();
 	delayMicroseconds(45);
 
-	DBG_PRINT(F("SRES Started,"));
-	cmdStrobe(CC1101_SRES);                                   // send reset
-	DBG_PRINT(F("POR Done,"));
+	DBG_PRINT(F("SRES started,"));
+	cmdStrobe(CC1101_SRES);                                // send reset
+	DBG_PRINT(F("POR done,"));
 	delay(10);
 
 	cc1101_Select();
 	DBG_PRINT(FPSTR(TXT_EEPROM)); 	DBG_PRINT(FPSTR(TXT_BLANK));	DBG_PRINT(FPSTR(TXT_READ));
-	wait_Miso();                                              // Wait until MISO goes low
+	wait_Miso();                                           // Wait until MISO goes low
 
 	sendSPI(0x00 | CC1101_WRITE_BURST);
-	for (uint8_t i = 0; i < sizeof(cc1101::initVal); i++) {   // write EEPROM value to cc1101
+	for (uint8_t i = 0; i < sizeof(cc1101::initVal); i++) {         // write EEPROM value to cc1101
 		sendSPI(EEPROM.read(EE_CC1101_CFG + i));
-		DBG_PRINT(".");
-		
+		DBG_PRINT('.');
 	}
+
 	cc1101_Deselect();
-	delayMicroseconds(10);            // ### todo: welcher Wert ist als delay sinnvoll? ###
+	delayMicroseconds(10);                                          // ### todo: welcher Wert ist als delay sinnvoll? ###
+
+	ccmode = ( (EEPROM.read(EE_CC1101_CFG + 18) ) & 0x70 ) >> 4;    // first read modulation direct from 0x12 MDMCFG2
 
-	writePatable();                   // write PatableArray to patable reg
+	writePatable();                                                 // write PatableArray to patable reg
 	DBG_PRINTLN(F("done"));
 
 	delay(1);
 	setReceiveMode();
 
 #endif
 }
+
+
+void cc1101::getRxFifo(uint16_t Boffs) {           // xFSK
+	uint8_t fifoBytes;
+	bool dup;                                      // true bei identischen Wiederholungen bei readRXFIFO
+
+	if (isHigh(PIN_RECEIVE)) {                     // wait for CC1100_FIFOTHR given bytes to arrive in FIFO
+		#ifdef PIN_LED_INVERSE
+			digitalWrite(PIN_LED, !blinkLED);
+		#else
+			digitalWrite(PIN_LED, blinkLED);
+		#endif
+
+/*
+ * 
+ * Ralf ( mode numbering == own selection )
```

Comment:
Was soll uns das Kommentar von Ralf hier bringen / sagen?

Reply:
Der Kommentar wurde erst vollständniskeitshalber übernommen. Über den Sinn bzw. Grund habe ich mir keine Gedanken gemacht weil es ein Kommentar ist ;-) Da müsste ich viele Kommentare im Code in Frage stellen *smile*
Gern nehme ich diesen heraus.

Reply:
Der Kommentar
>* Ralf ( mode numbering == own selection )

wurde von uns gesetzt. Diesen Verstehen wir natürlich, das war ein Fehler von mir.
Ganz simple,

Die Nummerierung von Ralf ist von Ihm gewählt. Ich vermute, je nach Implementierungsreihenfolge.

Der Kommentar
>* Sidey ( mode numbering == cc1101 data sheet setting numbering 0x12: MDMCFG2–Modem Configuration )

besagt aus, das deine bzw. unsere Version auf die Nummerierung sich auf das cc1101 Datasheet bezieht.
Somit haben wir bei dem Mode 3 auch das ASK/OOK wie im PDF benannt mit Mode 3.